client/lib/cart-values/cart-items.ts

```diff
@@ -11,40 +21,28 @@ import {
 	isGSuiteOrGoogleWorkspace,
 	isJetpackPlan,
 	isJetpackProduct,
+	isMonthlyProduct,
 	isNoAds,
-	isPlan,
-	isBlogger,
+	isP2Plus,
 	isPersonal,
+	isPlan,
 	isPremium,
-	isBusiness,
+	isRenewable,
 	isSiteRedirect,
 	isSpaceUpgrade,
+	isTitanMail,
+	isTrafficGuide,
 	isUnlimitedSpace,
 	isUnlimitedThemes,
 	isVideoPress,
-	isConciergeSession,
-	isTrafficGuide,
-	isTitanMail,
-	isP2Plus,
-	isMonthlyProduct,
-	isBiennially,
-	isRenewable,
-	getTermDuration,
-	getPlan,
-	isBloggerPlan,
-	isWpComFreePlan,
 	isWpComBloggerPlan,
-	isDIFMProduct,
+	isWpComFreePlan,
```

Comment:
I've just reordered those imports, and of course added the two from `calypso/lib/gsuite/constants`.